### PR TITLE
fix #4 : moved modules of viewer package from larray project to here

### LINKED ISSUE #4: Move viewer package from larray project to this one

### PR DESCRIPTION
**Pull request by alixdamman**
Need to activate Travis

### CODE REVIEW

**Comment by alixdamman** [comment]
linked to issue https://github.com/liam2/larray/issues/332 from larray

File: larray_editor/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, division, print_function
+
+try:
+    from larray_editor.api import *
+except ImportError:
+    def view(*args, **kwargs):
```
Comment (gdementen) [inline larray_editor/__init__.py]: wouldn't it be better to make larray_editor depend on pyqt/qtpy (in setup.py -- it is already the case in meta.yml) and avoid these dummy methods here? (we might want to keep something like this in larray though (at least temporarily).

**Comment by gdementen** [comment]
LGTM

**Comment by alixdamman** [comment]
Thanks for the review.

**Pull request merged**